packages/lambda-tiler/package.json

```diff
@@ -35,14 +37,21 @@
     ]
   },
   "scripts": {
-    "test": "ospec --globs 'build/**/*.test.js' --preload ../../scripts/test.before.js",
+    "test": "ospec --globs 'build/**/*.test.js' --ignore 'build/**/e2e.test.js' --preload ../../scripts/test.before.js",
+    "test:e2e": "ospec --globs 'build/**/e2e.test.js' --preload ../../scripts/test.before.js",
     "bundle": "./bundle.sh"
   },
   "devDependencies": {
     "@types/aws-lambda": "^8.10.43",
     "@types/express": "^4.17.8",
     "@types/node": "^14.11.2",
+    "@types/node-fetch": "^2.5.7",
+    "@types/pixelmatch": "^5.0.0",
+    "@types/pngjs": "^3.3.2",
+    "@types/sinon": "^9.0.8",
     "express": "^4.17.1",
-    "pretty-json-log": "^0.3.1"
+    "node-fetch": "^2.6.1",
+    "pretty-json-log": "^0.3.1",
+    "sinon": "^9.2.1"
```

Comment:
sinon is in the core `devDependencies` so I don't think they need to be explicitly included in each sub package